### ISSUE TITLE
Investigate error badkey and non_exists in Room

### ISSUE THREAD

**Issue by kanes115** [kanes115/what-if-backend#5]

**Comment by kanes115** [comment]
`not_exists` error appears when client asks about not existing room. It should trap that exit and answer with error (maybe print it on the screen - @Baksztag )

**Comment by kanes115** [comment]
Example crash:
```
[error] GenServer #PID<0.451.0> terminating
** (MatchError) no match of right hand side value: {:error, :not_exists}
    (what_if) lib/what_if/channels/room_channel.ex:99: WhatIf.RoomChannel.get_room_pid/1
    (what_if) lib/what_if/channels/room_channel.ex:27: WhatIf.RoomChannel.terminate/2
    (stdlib) gen_server.erl:648: :gen_server.try_terminate/3
    (stdlib) gen_server.erl:833: :gen_server.terminate/10
    (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
Last message: {:"$gen_cast", :close}
State: %Phoenix.Socket{assigns: %{user_id: "iEtHa6ntoZPPfBfy3VHHSn5reo42"}, channel: WhatIf.RoomChannel, channel_pid: #PID<0.451.0>, endpoint: WhatIf.Endpoint, handler: WhatIf.UserSocket, id: nil, joined: true, pubsub_server: WhatIf.PubSub, ref: nil, serializer: Phoenix.Transports.WebSocketSerializer, topic: "room:some room", transport: Phoenix.Transports.WebSocket, transport_name: :websocket, transport_pid: #PID<0.437.0>}
```

**Comment by kanes115** [comment]
Closed by #20
